geometry_msgs/Pose start_pose
geometry_msgs/Pose end_pose
nav_msgs/OccupancyGrid occupancy_grid
geometry_msgs/Pose[] final_path